android/src/main/java/com/getcapacitor/community/audio/AudioAsset.java

```diff
@@ -40,6 +46,26 @@
     }
   }
 
+  AudioAsset(
+    NativeAudio owner,
+    String assetId,
+    String url,
+    int audioChannelNum,
+    float volume
+  ) throws Exception {
+    this(owner, assetId);
+
+    if (audioChannelNum < 0) {
```

Comment:
when can we reach this scenario? Could we add a simple comment above this line?

Reply:
I don't know when or why or how. This has been done already by the plugin author. I guess that it could happen if the consumer of this plugin, in the web counterpart, sends a channel number < 0. And there's no sanitization process applied in the web layer, only in the native layer, so that's why it is done in here.